components/salsa-2022/src/storage.rs

```diff
@@ -164,6 +176,7 @@ where
     DB: HasJars,
 {
     fn drop(&mut self) {
+        drop(self.jars.take());
```

Comment:
This merits a comment -- I presume the idea is that the cvar is to be notified only after our ref onto `self.jars` is removed? (which makes sense)

Reply:
That's correct!